core/ledger/ledger_hash.go

```diff
@@ -105,6 +107,23 @@ func encodeJustify(buf *bytes.Buffer, block *pb.InternalBlock) error {
 	return nil
 }
 
+// VerifyMerkle
+func VerifyMerkle(block *pb.InternalBlock) error {
+	blockid := block.Blockid
+	merkleTree := MakeMerkleTree(block.Transactions)
+	if len(merkleTree) > 0 {
+		merkleRoot := merkleTree[len(merkleTree)-1]
+		blockMerkleRoot := global.F(block.MerkleRoot)
+		makeMerkleRoot := global.F(merkleRoot)
+		if blockMerkleRoot != makeMerkleRoot {
```

Comment:
No need to convert to a string for comparison, directly use `bytes.Equal(merkleRoot, block.MerkleRoot)` for comparison, reducing memory copy.

Reply:
ok